### PR TITLE
Use logical names and running a single query to get relationships

### PR DESCRIPTION
**Pull request by skg-marimuthu**
Use logical names and running a single query to get relationships for Azure SQL

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/tests/Microsoft.PowerFx.Connectors.Tests.Shared/PowerPlatformConnectorTests.cs [hunk 1]
```diff
@@ -21,9 +21,11 @@
 using Microsoft.PowerFx.Core.Types;
 using Microsoft.PowerFx.Functions;
 using Microsoft.PowerFx.Intellisense;
+using Microsoft.PowerFx.Syntax;
 using Microsoft.PowerFx.Types;
 using Xunit;
 using Xunit.Abstractions;
+using static System.Net.Mime.MediaTypeNames;
```
Comment (LucGenetier) [inline src/tests/Microsoft.PowerFx.Connectors.Tests.Shared/PowerPlatformConnectorTests.cs:28]: Really? :)

**Comment by LucGenetier** [comment]
>         public async Task SQL_GetRelationships()

Do you confirm you've run this test w/ a like instance?
(alongside with recorded network results)

---
Refers to: src/tests/Microsoft.PowerFx.Connectors.Tests.Shared/PowerPlatformConnectorTests.cs:1380 in c33bfde. [](commit_id = c33bfdeb84b9585d8e2396ccee31840edf8b9b11, deletion_comment = False)

**Comment by skg-marimuthu** [comment]
> > ```
> >     public async Task SQL_GetRelationships()
> > ```
> 
> Do you confirm you've run this test w/ a like instance? (alongside with recorded network results)
> 
> Refers to: src/tests/Microsoft.PowerFx.Connectors.Tests.Shared/PowerPlatformConnectorTests.cs:1380 in [c33bfde](https://github.com/microsoft/Power-Fx/commit/c33bfdeb84b9585d8e2396ccee31840edf8b9b11). [](commit_id = [c33bfde](https://github.com/microsoft/Power-Fx/commit/c33bfdeb84b9585d8e2396ccee31840edf8b9b11), deletion_comment = False)

Yes, I have tested with the actual SQL instance and updated the sample relationships specific to required table

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by LucGenetier — APPROVED** [review]
:shipit:

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**